parsers/ENTE.py

```diff
@@ -25,6 +27,10 @@
 }
 
 
+def floor_to_minute(dt: datetime) -> datetime:
+    return dt.replace(second=0, microsecond=0)
```

Comment:
This is pretty nice, maybe we should create a common set of helper functions for this in the future.

Reply:
Yes, if other parsers are (or will) perform this logic, then it would make sense to move this `floor_to_minute` function to a common shared module. There is already a `parsers/lib/utils.py` file, which might be a good candidate to move this function to.